packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -283,6 +272,11 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 		},
 	},
 	actions: {
+		getPinDataSize(pinData: Record<string, unknown> = {}): number {
+			return Object.values(pinData).reduce<number>((acc, value) => {
```

Comment:
```suggestion
			return Object.values(pinData).reduce(acc, value) => {
```

Reply:
Weirdly enough, typescript is unable to infer the number type here without the explicit type generic.

Reply:
Works on my end, might be a Webstorm thing, super minor though.

<img width="831" alt="Capture 2023-11-01 at 17 46 29@2x" src="https://github.com/n8n-io/n8n/assets/44588767/482b57a6-5e74-423b-86ca-026105f36ccb">